qemu/tests/numa_memdev_options.py

```diff
@@ -82,7 +82,7 @@ def check_memory_in_procfs(test, params, vm):
         if mem_path and (mem_path not in smaps):
             test.fail("memdev = %s: mem-path '%s' is not in smaps '%s'!"
                       % (mem_dev, mem_path, smaps))
-        mem_start = smaps.split('-')[0]
+        mem_start = re.findall('^([0-9a-fA-F]+)-', smaps, re.M)[0]
```

Comment:
Well, this change can make it work, however i think the way to get `smaps` can be improved too since the output include to many invalid info now.
If i understand correctly, it(L78) can be limited to:
```
        smaps = process.system_output("grep -B1 -E 'Size:\s+%d' /proc/%d/smaps"
                                      % (mem_size, qemu_pid))
```

For example:
```
# grep -B1 -E 'Size:\s+1048576' /proc/96383/smaps
7f8e7be00000-7f8ebbe00000 rw-p 00000000 00:2d 937596                     /mnt/kvm_hugepage/qemu_back_mem._objects_mem-mem0.2qrwme (deleted)
Size:            1048576 kB
```